fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/api/resource/conditions/v1/DefaultResourceConditions.java

```diff
@@ -146,6 +148,17 @@ public static <T> ConditionJsonProvider tagsPopulated(TagKey<T>... tags) {
 		return ResourceConditionsImpl.tagsPopulated(TAGS_POPULATED, true, tags);
 	}
 
+	/**
+	 * Creates a condition that returns true if the passed feature is enabled.
+	 * @param feature the feature to check for
+	 *
+	 * @apiNote This condition's ID is {@code fabric:feature_enabled}, and takes one property:
+	 * {@code feature}, which is the ID of the feature flag to check.
+	 */
+	public static ConditionJsonProvider featureEnabled(FeatureFlag feature) {
```

Comment:
I would like to see this condition renamed to `features_enabled` and accept multiple features (like the others above).